### ISSUE TITLE
Enhance readability of "choco outdated" output

### ISSUE THREAD

**Issue by ahorn42** [chocolatey/choco#2889]
## Is Your Feature Request Related To A Problem? Please describe.

The output of the command `choco outdated` is not very easy to read as human, as the values are simple concatenated but not aligned, like in a table for example.

## Describe The Solution. Why is it needed?

The output could be formatted as table to enhance readability.
See the following screenshot for a proposal (top is old output, bottom is enhanced output as table)
![image](https://user-images.githubusercontent.com/5106205/200339781-6b872578-278c-49b0-b82a-e2150f6fec5f.png)

## Additional Context.

I already implemented a draft on how to handle this. I will link it in the comments.

## Related Issues

N/A -> no related issues (at least I found none)

**Comment by pauby** [comment]
Duplicate of #2773 ?

**Comment by pauby** [comment]
👍 Let me go ahead and mark this as a duplicate and close it.

**Comment by ahorn42** [comment]
So I am not sure if you are interested in my suggestion, if there is a plan to migrate to Spectre at some point? If you want to use it as an intermediate improvement I am happy to prepare it for a merge request, otherwise I am eager waiting for the Spectre solution ;)

**Comment by pauby** [comment]
We will have to be at v3.0.0 of Chocolatey CLI before we could consider using Spectre.

@gep13 you've been eyeing up Spectre support. Can you confirm?

**Comment by gep13** [comment]
Yes, my longer term plan is to introduce Spectre.Console (or something very similar) to allow the control of the current "UI" for Chocolatey CLI.  There are a number of areas that can be improved, including the output from:

- choco outdated
- choco search
- choco source
- choco apikey
- choco help
- etc

I think I would prefer to see this done as a block piece of work, and port everything to Spectre.Console, rather than do work in changing the current output, only to have it changed again.

Thoughts?
